doc/modelling/reaction/index.rst

```diff
@@ -9,7 +9,7 @@ reaction mechanism.
 CADET features one reaction type:
 
  - :ref:`mass_action_law_model` (most commonly used)
- - :ref:`michaelis_menten_kinetics_model`
+ - :ref:`michaelis_menten_kinetics_model` (for enzyme kinetics)
```

Comment:
A since the equation is analogue to the [Monod Equation](https://en.wikipedia.org/wiki/Monod_equation) for cell growth, we should also mention this here. What do you think @lieres